src/main/java/hudson/plugins/git/GitTagAction.java

```diff
@@ -185,15 +185,10 @@ void scheduleTagCreation(Map<String, String> newTags, String comment) throws IOE
      */
     public final class TagWorkerThread extends TaskThread {
         private final Map<String, String> tagSet;
-        /**
-         * If the user provided a separate credential, this object represents that.
-         */
-        private final String comment;
```

Comment:
So the `TagWorkerThread` class isn't static. So it maybe could be serialized together with the action?
I might be turning things around in my head. It's still early in the morning :)

Reply:
When the outer class is serialized (for example, to apply a tag to an agent workspace), the inner class will be serialized as well.  That is a good thing in this case, since the inner class does the work of applying the tag to the workspace.

At least, that's what I'm interpreting from the [stackoverflow article](https://stackoverflow.com/questions/10797754/serializing-a-inner-class-instance) that I read.

@rsandell does that cause you worry?  Is there something that I should test more deeply to assure this does not break something?